src/agent/driver-test-runner.js

```diff
@@ -119,21 +119,29 @@ export class DriverTestRunner {
         if (!desiredResponse) {
           throw new Error(`Unknown command settings for NVDA "${settings}"`);
         }
-        const MODE_SWITCH_SPEECH_TIMEOUT = 500;
-        let speechResponse;
+        // try to keep this short because it runs up to twice for every test
+        // have seen occasional timeout/misses at 500ms
```

Comment:
This is a little vague (and colloquial besides). Have you observed delays longer than 500ms in the GitHub Actions environment? During local testing?

Reply:
Local test

Reply:
On a vm

Reply:
Thanks. I've clarified those comments.